WordPress/Classes/ViewRelated/NUX/LoginEpilogueTableView.swift

```diff
@@ -45,16 +46,11 @@ class LoginEpilogueTableView: UITableViewController {
             guard let cell = tableView.dequeueReusableCell(withIdentifier: "userInfo") as? LoginEpilogueUserInfoCell else {
                 fatalError("Failed to get a user info cell")
             }
-            guard let account = defaultAccount() else {
```

Comment:
`defaultAccount` is defined in this class and was copied from elsewhere. I see you've replaced it with `service.defaultWordPressComAccount` so let's delete `defaultAccount` here.